plugins/save/plugin.js

```diff
@@ -44,8 +44,9 @@
 			if ( editor.elementMode != CKEDITOR.ELEMENT_MODE_REPLACE )
 				return;
 
+			saveCmd.startDisabled = !( editor.element.$.form );
```

Comment:
This property should be added to `command`, not `saveCmd`. `saveCmd` is a template to create commands in all editors and `command` is an instance of that command in particular editor. We shouldn't modify the template, but the instance.